docs/source/examples/syncing-code-artifacts.rst

```diff
@@ -18,49 +18,111 @@ Here, "upload" means uploading files from your local machine (or a cloud object
 storage) to a SkyPilot cluster, while "download" means the reverse direction.  The same
 mechanisms work for both files and directories.
 
-.. _upload-code-and-project-files:
+.. _upload-code-and-project-files-git:
 
-Uploading code and project files
---------------------------------------
-SkyPilot **automatically syncs a local working directory to a cluster** on every
-:code:`sky launch` and :code:`sky exec`.  The workdir contains a project's
-code and other files, and is typically a Git folder.
+Uploading code and project files or cloning a git repository
```

Comment:
Ditto, `uploading` and `cloning` feels like talking too much about our implementation, can we simplify the concept to "sync the workdir from local dir or remote dir`?

Reply:
Updated.